index.html

```diff
@@ -9,25 +9,65 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">   
     <link href="https://fonts.googleapis.com/css2?family=DM+Sans&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Pacifico&display=swap" rel="stylesheet"> 
     <title>Awesome Books</title>
 </head>
 <body>
-    <h1 class="title">All awesome books</h1>
+    <main>
+        <header>
+            <div class="menu-container">
+                <h2>Awesome Books</h2>
+                <nav class="principal-menu">
+                    <ul>
+                        <li><a class="list-link" href="#list">List</a></li>
+                        <li class="middle-nav"><a class="add-book-link" href="#add-book">New</a></li>
+                        <li><a class="contact-link" href="#contact">Contact</a></li>
+                    </ul>
+                </nav>
+            </div>
+            <p class="date"></p>
+          </header>
+    
+        
+    
+        <section id="list" >
+            <h1 class="title">All awesome books</h1>
+            <div class="books-container"></div>
+        </section>
+    
+    
+        <section id="add-book" class="new-book-container">
+            <h2 class="new-book-title">Add a new book</h2>
+            <form class="add-book-form" action="index.html">
+                <input class="title-input" type="text" placeholder="Title">
+                <input class="author-input" type="text" placeholder="Author">
+                <input class="add-book-btn" type="button" value="Add">
+            </form>
```

Comment:
## [OPTIONAL]

- [ ] Kindly I would suggest you render and display the list of books when the form of adding an new book is submitted or display a notification text saying that the book has been created successfully.